brink/the-bitcoin-development-podcast/code-review-and-bip324.md

```diff
@@ -372,24 +377,23 @@ You familiarize yourself with the topic by reading the BIP and maybe looking at
 
 Sebastian Falbesoner: 00:31:46
 
-What I first did back then still was maintaining the PRs. So I just first built the main PR that includes all the sub-PRs and ran a node just to also have something available already and playing a bit around what the end product would look like and then iteratively I would look at those sub PRS and No, I didn't look at the SEC be parts until recently It always was appeared very scary for me all the cryptography and math stuff but I also ended up doing a little review there.
-I was lucky enough to join a hangout at the last CoreDev meeting in Ireland where Peter gave an introduction to the LibSecP library and he specifically presented the PR 1129, which is the Alligator Swift part.
+What I first did back then still Dhruv was maintaining the PRs.
+So I just first built the main PR that includes all the sub-PRs and ran a node just to also have something available already and playing a bit around what the end product would look like and then iteratively I would look at those sub-PRS and no, I didn't look at the SECP parts until recently It always was appeared very scary for me all those cryptography and math stuff but I also ended up doing a little review there.
+I was lucky enough to join a hangout at the last CoreDev meeting in Ireland where Peter gave an introduction to the `libsecp` library and he specifically presented the PR1129, which is the Alligator Swift part.
 So I got a little deeper into that through that.
 
```

Comment:
reviewing transcripts is an opportunity to learn, so what you could here is to go on your favorite search engine and search for "bitcoin libsecp github PR 1129", which would eventually let you to the PR: https://github.com/bitcoin-core/secp256k1/pull/1129

and there you could see that it's "ElligatorSwift"
```suggestion
I was lucky enough to join a hangout at the last CoreDev meeting in Ireland where Peter gave an introduction to the `libsecp` library and he specifically presented the PR1129, which is the ElligatorSwift part.
```